.gitlab/generate-tracer.php

```diff
@@ -513,7 +557,7 @@ function before_script_steps() {
 $services["deferred_loading"] = "mysql";
 $services["deferred_loadin"] = "redis";
 $services["pdo"] = "mysql";
-$services["kafk"] = ["kafka", "zookeeper"];
+$services["kafka"] = ["kafka", "zookeeper"];  // Overwrite auto-generated entry
```

Comment:
Nice catch, seems like this should have had been a problem in the past, but it wasn't? 🤯